README.rst

```diff
@@ -124,18 +124,33 @@ sites)::
   ./manage.py lms collectstatic
   ./manage.py cms collectstatic
 
+Setup Studio SSO for Development::
+
+  ./manage.py lms manage_user studio_worker example@example.com --unusable-password
+  # DO NOT DO THIS IN PRODUCTION it will make your auth insecure
+  ./manage.py lms create_dot_application --grant-type authorization-code --skip-authorization --redirect-uris "http://localhost:18010/complete/edx-oauth2/" --scopes "user_id" studio-sso studio_worker --client-id studio-sso-key --client-secret studio-sso-secret
```

Comment:
```suggestion
Set up CMS SSO (for Development)::

  ./manage.py lms manage_user cms example@example.com --unusable-password
  # DO NOT DO THIS IN PRODUCTION. It will make your auth insecure.
  ./manage.py lms create_dot_application \
      --grant-type authorization-code\
	  --skip-authorization \
	  --redirect-uris "http://localhost:18010/complete/edx-oauth2/" \
	  --scopes "user_id" cms-sso cms \
	  --client-id cms-sso \
	  --client-secret cms-sso-secret
```

* setup is two words when it's a verb
* the service is called CMS, not Studio
* use parentheses to show that it's one step with two variations (dev vs prod) rather than two steps that you're supposed to perform in series
* break up long shell commands for readability
* Devstack calls the user `studio_worker`, but Tutor calls it user `cms`. I think the ideal name would be `cms_worker`, but then we're inventing a third default name, so my instinct is just to name the user `cms`

Reply:
EDIT: the `studio-sso-id -> cms-sso` and `studio-sso-secret -> cms-sso-secret` suggestions only make sense if you're doing dev with production.py. If you're using devstack.py, it makes sense that you'd just want to run with the defaults that Devstack gives you.